google-maps/src/implementation.ts

```diff
@@ -105,13 +105,23 @@ export interface OnScrollArgs {
   };
 }
 
+export interface EnableClusteringArgs {
+  id: string;
+  /**
+   * The minimum number of markers that can be clustered together.
+   *
+   * @default 4
+   */
```

Comment:
The way google-maps plugin is built won't show this information to the users nor appear on the README.
For it to be visible it should be moved to the `enableClustering` method definition in `map.ts` file.

Reply:
Ok, I moved it for this PR. However, I'm also going to create a separate PR to copy the docs on the method definitions to the GoogleMapsInterface since the README is missing descriptions for all of the methods.